solidity/slither.db.json

```diff
@@ -0,0 +1 @@
KeepVendorImplV1.sol", "is_dependency": false, "lines": [25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127, 128, 129, 130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146, 147, 148, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160, 161, 162, 163, 164, 165, 166, 167, 168, 169, 170, 171], "starting_column": 1, "ending_column": 2}}}, "additional_fields": {"target": "variable", "convention": "mixedCase"}}], "description": "Variable BondedECDSAKeepVendorImplV1._initialized (BondedECDSAKeepVendorImplV1.sol#30) is not in mixedCase\n", "markdown": "Variable [BondedECDSAKeepVendorImplV1._initialized](contracts/BondedECDSAKeepVendorImplV1.sol#L30) is not in mixedCase\n", "id": "f8600662c978fdd723a6be2887c211f6311bc8dcd7747066f7af0d834e0691c5", "check": "naming-convention", "impact": "Informational", "confidence": "High"}, {"elements": [{"type": "function", "name": "__beaconCallback", "source_mapping": {"start": 3697, "length": 122, "filename_used": "/home/lukasz/go/src/github.com/keep-network/keep-ecdsa/solidity/contracts/GroupSelectionSeed.sol", "filename_relative": "contracts/GroupSelectionSeed.sol", "filename_absolute": "/home/lukasz/go/src/github.com/keep-network/keep-ecdsa/solidity/contracts/GroupSelectionSeed.sol", "filename_short": "GroupSelectionSeed.sol", "is_dependency": false, "lines": [60, 61, 62], "starting_column": 5, "ending_column": 6}, "type_specific_fields": {"parent": {"type": "contract", "name": "GroupSelectionSeed", "source_mapping": {"start": 2054, "length": 5237, "filename_used": "/home/lukasz/go/src/github.com/keep-network/keep-ecdsa/solidity/contracts/GroupSelectionSeed.sol", "filename_relative": "contracts/GroupSelectionSeed.sol", "filename_absolute": "/home/lukasz/go/src/github.com/keep-network/keep-ecdsa/solidity/contracts/GroupSelectionSeed.sol", "filename_short": "GroupSelectionSeed.sol", "is_dependency": false, "lines": [22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127, 128, 129, 130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144], "starting_column": 1, "ending_column": 2}}, "signature": "__beaconCallback(uint256)"}, "additional_fields": {"target": "function", "convention": "mixedCase"}}
```

Comment:
Is it possible to have this file formatted in the repository? In case we change the code and have to add/modify some ignore, doing a review of ignores will not be possible having them all in a single line.

(Just flagging it, I am aware it's just a draft now.)

Reply:
I'll see what can we do here.

Reply:
Slither doesn't provide formatting of this file out of the box. I added the `pretty-format-json` pre commit hook to provide formatting.